Makefile

```diff
@@ -5,11 +5,28 @@ querier := robots/cmd/release-querier
 kubevirtci := robots/cmd/kubevirtci-bumper
 bazelbin := bazelisk
 
-.PHONY: all clean deps-update gazelle-update-repos update-labels $(limiter) $(flake-report-writer) $(querier) $(kubevirtci) $(flake-issue-creator)
+ifndef GOLANGCI_LINT_VERSION
+	GOLANGCI_LINT_VERSION=v1.62.2
+	export GOLANGCI_LINT_VERSION
+endif
+ifndef ARTIFACTS
+	ARTIFACTS=/tmp/artifacts
+	export ARTIFACTS
+endif
+ifndef COVERAGE_OUTPUT_PATH
+	COVERAGE_OUTPUT_PATH=${ARTIFACTS}/coverage.html
+	export COVERAGE_OUTPUT_PATH
+endif
+
+.PHONY: all make-artifacts-dir clean deps-update gazelle-update-repos update-labels install-metrics-binaries lint $(limiter) $(flake-report-writer) $(querier) $(kubevirtci) $(flake-issue-creator)
 all: deps-update $(limiter) $(flake-report-writer) $(querier) $(kubevirtci) $(flake-issue-creator)
 
-clean:
+make-artifacts-dir:
```

Comment:
As this is only used by the coverage target - can it be included there? rather than having its own target?